clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -1310,7 +1315,9 @@ raw2trace_t::aggregate_and_write_schedule_files()
     for (auto &keyval : cpu2sched) {
         std::sort(keyval.second.begin(), keyval.second.end(),
                   [](const schedule_entry_t &l, const schedule_entry_t &r) {
-                      return l.timestamp < r.timestamp;
+                      if (l.timestamp != r.timestamp)
+                          return l.timestamp < r.timestamp;
+                      return l.start_instruction < r.start_instruction;
```

Comment:
I would again use the same lambda to make it clear it's the same comparison and avoid further errors where someone updates one and not the other (which apparently happened recently right?)

Isn't this wrong since it's not checking the thread?  Yet another reason to use the same lambda.

Reply:
Done.